osi_referenceline.proto

```diff
@@ -87,31 +111,57 @@ message ReferenceLine
     // ## Adding T coordinates
     //
     // To describe points that are not directly on the polyline, a T coordinate
-    // is added. T is the signed 2D distance (i.e. <code>hypot(A.X-B.X,
-    // A.Y-B.Y)</code>, if A and B are the two points) between the point to
-    // describe and the nearest point on the polyline (this point might either
-    // be on a line segment or at an edge between two line segments). The
-    // distance is positive if the point is left of the polyline (in definition
-    // direction), negative if it is right of it.
-    // The S position of such a point outside the reference line is the same as
-    // the S value of the nearest point on the polyline.
-    //
-    // Notes:
-    // - The "nearest point on the polyline" is determined in 3D (even if the
-    //   resulting T value is only the 2D distance), in order to choose the
-    //   correct point for 3D curves (think reference lines for roads in parking
-    //   decks).
-    // - If there are several "nearest points", the one with the smallest S
-    //   coordinate on the polyline is chosen.
+    // is added. T is the signed 2D distance between the point to describe (P)
+    // and a projected point (P_proj) on the polyline. There are two ways of
+    // defining this point, depending on the ReferenceLine type.
+    //
+    // ## Nearest point (TYPE_POLYLINE)
+    //
+    // The projection point is the nearest point on the polyline (this point might
```

Comment:
This seems far too short (e.g. it doesn't mention which side is positive). I guess the goal here is to keep backward compatibility. Then either the old documentation should be kept, or common parts between the two modes should be in a common place of the documentation. E.g. now only TYPE_POLYLINE_WITH_T_AXIS mentions which side is positive, but this applies to both modes.

Reply:
re-added old documentation